packages/block-library/src/query-title/variations.js

```diff
@@ -14,6 +14,37 @@ const variations = [
 		icon: archiveTitle,
 		attributes: {
 			type: 'archive',
+			content: __( 'Archive' ),
+		},
+		scope: [ 'inserter' ],
+	},
+	{
+		name: 'search-title',
+		title: __( 'Search Title' ),
```

Comment:
Do these need to be variations or could we just infer from context what to display? As a template author, what happens if I only have `index.html` with archive title added?

Reply:
If we're able to infer what to display from the page context that would work better I think. This would be much quicker for creating and editing templates. I'll look into this, thanks!